docs/setup/compute-ntp.md

```diff
@@ -35,6 +34,12 @@ If your Create® 3 and compute board have an internet connection, this is not re
 
         sudo service chrony restart
 
+1. Log into the Create® 3 web application and modify the NTP sources (as of Jul 2023 under "Beta Features") to add the following: 
+
+        server 192.168.186.3 iburst
```

Comment:
This should be in the ntp.conf file by default; maybe change line 37 to read "...to add the following, if it does not exist"

Reply:
Good call out! Not sure why my robot didn't have that in there - maybe I messed with it at some point. Done!